domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -2840,14 +2840,14 @@ public void streamGetExceptionInHandler(TestContext context) throws IOException,
         });
         sr.endHandler(x -> {
           events.append("[endHandler]");
+          async.complete();
         });
         sr.exceptionHandler(x -> {
           events.append("[exception]");
```

Comment:
Maybe the exceptionHandler can be moved above endHandler, so the code order aligns with the testing logic better?

Reply:
Reordered.